new_project/readme.md

```diff
@@ -0,0 +1,88 @@
+# Github Release Process
+
+This directory delineates the process, guidelines, and requirements for releasing 
+new projects to the [niosh-mining github page](https://github.com/niosh-mining).
+
+# Pre-Review Requirements
+
+Before starting the github release approval process, please ensure the candidate code satisfies the
+following requirements:
+
+    1. The code is stored in a git repository
+    2. The repo does not include any [sensitive data](#Data) (consider removing old history).
+    3. The repo includes [documentation appropriate to the maturity of the software](#Documentation).
+    4. The code includes a [testing procedure](#Testing).
+    5. The code includes an [OSI approved open-source liscense](https://opensource.org/licenses).
+    6. The code includes a [version string and a versioning system](#Versioning)
+    7. The code has been uploaded to the niosh-mining github page as a **private** repo.
```

Comment:
Is the person who wants to upload the repo able to make it public without anyone's intervention? Or does it have to be private by default?

Reply:
It can be made public. I don't know if there are any administrative controls which can prevent creating public repos but I will look into it.

Reply:
It looks like we don't have the right organization account to restrict public repo creation. We need  GitHub Enterprise Cloud: https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/restricting-repository-creation-in-your-organization